src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -678,6 +680,53 @@ class ShaderCache : public IECore::RefCounted
 
 IE_CORE_DECLAREPTR( ShaderCache )
 
+class NodeVectorCache : public IECore::RefCounted
+{
+
+	public :
+
+		const std::vector<AtNode *> &get( IECore::ConstStringVectorDataPtr nodeNamesData, const std::string &prefix = "" )
```

Comment:
Just `const StringVectorData *nodeNamesData`, since this function doesn't need any ownership.

Reply:
Of course, thanks. Merged my changes into 16470e0.